dotcom-rendering/src/components/AdSlot.tsx

```diff
@@ -317,16 +323,59 @@ export const AdSlot = ({
 					);
 				}
 				case ArticleDisplay.Standard: {
-					return (
+					return isLiveblog ? (
```

Comment:
Just to confirm, is the idea here that we only want to hydrate the top right ad slot island on a liveblog so that'll we can use the `useAB` hook, and avoid doing so on other layouts?

If so, I wonder if we can make that more explicit? Otherwise, I have a suggestion in another comment that might mean we can avoid hydrating the right hand ad slot at all.

Reply:
Yep, will follow your suggestion in the other comment to have all the ab test logic is one place and remove the island behaviour from `TopRightAdSlot.tsx`.